bakery/example/idservice/idservice.go

```diff
@@ -59,25 +59,25 @@ func New(p Params) (http.Handler, error) {
 	}
 	mux := http.NewServeMux()
```

Comment:
I think perhaps this could use an httprouter mux. We couldn't do that in httpbakery because the
http mux is part of the public API but I think it would make sense here.